adhocracy4/projects/models.py

```diff
@@ -83,6 +89,12 @@ class Project(base.TimeStampedModel):
         ),
         upload_to='projects/tiles',
         blank=True)
+    tile_image_copyright = models.CharField(
```

Comment:
Won't at lease a function here be nice, that takes the other field and generates a copy right field from it?

```
def make_copyright_field(imagefield):
    return models.CharField(
      max_length=120,
      verbose_name=imagefield.verbose_name + ' ' + _('copyright')
     ...
   )
```

Reply:
Added a custom copyright field to set these defaults.